db-client-java/src/main/java/com/eventstore/dbclient/AbstractPersistentSubscriptionSettingsBuilder.java

```diff
@@ -0,0 +1,141 @@
+package com.eventstore.dbclient;
+
+public class AbstractPersistentSubscriptionSettingsBuilder<T> {
+    protected int checkpointAfterMs;
+    protected boolean extraStatistics;
+    protected boolean resolveLinks;
+    protected boolean fromStart;
+    protected boolean fromEnd;
+    protected int historyBufferSize;
+    protected int liveBufferSize;
+    protected int maxCheckpointCount;
+    protected int maxRetryCount;
+    protected int maxSubscriberCount;
+    protected int messageTimeoutMs;
+    protected int minCheckpointCount;
+    protected int readBatchSize;
+    protected ConsumerStrategy strategy;
+
+    public AbstractPersistentSubscriptionSettingsBuilder() {
+        checkpointAfterMs = 2_000;
+        resolveLinks = false;
+        extraStatistics = false;
+        fromStart = false;
+        fromEnd = false;
+        messageTimeoutMs = 30_000;
+        maxRetryCount = 10;
+        minCheckpointCount = 10;
+        maxCheckpointCount = 1_000;
+        maxSubscriberCount = 0;
+        liveBufferSize = 500;
+        readBatchSize = 20;
+        historyBufferSize = 500;
+        strategy = ConsumerStrategy.RoundRobin;
+    }
+
+    public AbstractPersistentSubscriptionSettingsBuilder(AbstractPersistentSubscriptionSettings settings) {
+        checkpointAfterMs = settings.getCheckpointAfterMs();
+        resolveLinks = settings.isResolveLinks();
+        extraStatistics = settings.isExtraStatistics();
+        fromStart = settings.getFromStart();
+        fromEnd = settings.getFromEnd();
+        messageTimeoutMs = settings.getMessageTimeoutMs();
+        maxRetryCount = settings.getMaxRetryCount();
+        minCheckpointCount = settings.getMinCheckpointCount();
+        maxCheckpointCount = settings.getMaxCheckpointCount();
+        maxSubscriberCount = settings.getMaxSubscriberCount();
+        liveBufferSize = settings.getLiveBufferSize();
+        readBatchSize = settings.getReadBatchSize();
+        historyBufferSize = settings.getHistoryBufferSize();
+        strategy = settings.getStrategy();
+    }
+
+    public T enableLinkResolution() {
```

Comment:
**FIX:** ResolveLinkTos naming is not aligned with other options. See more e.g. in [OptionsWithResolveLinkTosBase](https://github.com/EventStore/EventStoreDB-Client-Java/blob/trunk/db-client-java/src/main/java/com/eventstore/dbclient/OptionsWithResolveLinkTosBase.java#L19).

Reply:
This was already in-place, deprecated it and linked to new `resolveLinkTos()`.

Reply:
This was already in-place, deprecated it and linked to new method.